cmd/state/internal/cmdtree/builds.go

```diff
@@ -19,9 +20,19 @@ func newBuildsCommand(prime *primer.Values) *captain.Command {
 		[]*captain.Flag{
 			{
 				Name:        "all",
-				Description: "List all builds, including individual package artifacts",
+				Description: locale.Tl("builds_flags_all_description", "List all builds, including individual package artifacts"),
 				Value:       &params.All,
 			},
+			{
+				Name:        "namespace",
+				Description: locale.Tl("builds_flags_namespace_description", "The namespace of the project to inspect builds for"),
+				Value:       params.Namespace,
```

Comment:
```suggestion
				Value:       &params.Namespace,
```

I guess this is why line 13 had the empty struct. But you shouldn't need that.